assets/js/script.js

```diff
@@ -150,48 +168,65 @@ function render() {
   $('.js-tab').html('')
   $('.js-preview').html('')
   $('.js-tab').removeClass('active')
+  $('.js-tab').parent().removeClass('active')
```

Comment:
so, `.parent()` can be a fragile construct that breaks unexpectedly when the DOM changes. I would recommend tagging the element you're trying to reference with a `js-` so that a future developer knows the js depends on that element being where it is. It looks like that's already done in this case, so this would be chill:

`$('.js-tab-back').removeClass('active')`